URL-based headers not working with CL

[example](https://godbolt.org/g/oXowPQ)
This happens with 32 and 64-bit. Compiler output:

```
example.cpp
<source>(1): error C2019: expected preprocessor directive, found '1'
<source>(5): error C2019: expected preprocessor directive, found '1'
Microsoft (R) C/C++ Optimizing Compiler Version 19.10.25017 for x86
Copyright (C) Microsoft Corporation.  All rights reserved.
Compiler exited with result code 2
```

--- a/static/compiler-service.js
+++ b/static/compiler-service.js
@@ -115,7 +115,7 @@ define(function (require) {
             if (match) {
                 promises.push(new Promise(function (resolve, reject) {
                     var req = $.get(match[1], function (data) {
-                        data = '# 1 "' + match[1] + '"\n' + data + '\n\n# ' +
+                        data = '#line 1 "' + match[1] + '"\n' + data + '\n\n#line ' +
                             (lineNumZeroBased + 1) + ' "<stdin>"\n';
 
                         lines[lineNumZeroBased] = data;

